### ISSUE TITLE
test installer for the asdf-vm version manager

### ISSUE THREAD

**Issue by tomdavidson** [npryce/adr-tools#42]
I created a [adr-tools plugin](https://gitlab.com/td7x/asdf/adr-tools) for [asdf-vm](https://github.com/asdf-vm/asdf) and have an opened [PR](https://github.com/asdf-vm/asdf-plugins/pull/20) to register it.

asdf gives osx and linux users (ci jobs) a 'universal' path to managing tool versions. Anyone care to test it out? especially osx users? Here is a quicky, but read the asdf README.md too. Once the asdf-plugin PR is approved, the git urls will not be needed:

```sh
$ asdf plugin-add adr-tools https://gitlab.com/td7x/asdf/adr-tools
$ asdf list-all adr-tools
$ asdf install adr-tools 2.0.1
$ asdf (global | local) adr-tools 2.0.1
$ adr
```

It would be nice to have others try it out and make sure it works beyond my pipeline & computer.

**Comment by npryce** [comment]
It's been merged into the asdf-vm plugins project, so I assume it works ok.
